ui/js/signup.js

```diff
@@ -239,6 +249,9 @@ require(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
                 },
                 'password_repeat': {
                     'equalTo': '#signup-createaccount-password'
+                },
+                'email': {
+                    'emaildomain': true
```

Comment:
Rather than adding this here, you can just add `emaildomain` as a property to the email field